### ISSUE TITLE
Buttons on exhibition guides have insufficient colour contrast

### ISSUE THREAD

**Issue by davidpmccormick** [wellcomecollection/wellcomecollection.org#8806]
'Read full transcript' buttons inside context sections (those with a beige background) don't have a high enough contrast.

![Image](https://user-images.githubusercontent.com/1394592/199478944-f76896ed-7e58-4588-b19e-78133deffeb3.png)

https://dash.wellcomecollection.org/pa11y/

**Comment by rcantin-w** [comment]
I had that elsewhere and think I went for `charcoalTransparentCharcoal` instead
